tests/specs/nodeRewardsProgramModule.spec.js

```diff
@@ -0,0 +1,576 @@
+            }]
+
+            expect(ctrl.accounts).toEqual(expectedAccounts);
+            expect(ctrl.selectedAccount).toEqual(expectedAccounts[0]);
+            expect(ctrl.balance).toEqual(expectedAccounts[0].formatBalance);
+            expect(ctrl.formData.isMultisig).toEqual(expectedAccounts[0].isMultisig);
+            expect(ctrl.formData.mainPublicKey).toEqual(expectedAccounts[0].publicKey);
+        })
+    })
+
+    describe('setSelectedAccountInfo', () => {
+        const runBasicSetSelectedAccountInfoTests = (selectedAccount) => {
+            // Arrange:
+            const ctrl = $controller('NodeRewardsProgramCtrl', {
+                $scope: $rootScope.$new()
+            });
+
+            ctrl.selectedAccount = selectedAccount;
+
+            // Act:
+            ctrl.setSelectedAccountInfo();
+
+            // Assert:
+            expect(ctrl.balance).toEqual(selectedAccount.formatBalance);
+            expect(ctrl.formData.isMultisig).toEqual(selectedAccount.isMultisig);
+            expect(ctrl.formData.mainPublicKey).toEqual(selectedAccount.publicKey);
+        }
+
+        it('set normal account info to form data and balance', () => {
+            // Arrange:
+            const mockSelectedAccount = {
+                publicKey: 'ca37a2ee96b94fd5e7879105733c97a21ffc2c00d961b2995a59a82463708e6a',
+                formatBalance: '0.000010',
+                isMultisig: false
+            }
+
+            runBasicSetSelectedAccountInfoTests(mockSelectedAccount);
+        })
+
+        it('set multisig account info to form data and balance', () => {
+            // Arrange:
+            const mockSelectedMultisigAccount = {
+                publicKey: 'ca37a2ee96b94fd5e7879105733c97a21ffc2c00d961b2995a59a82463708e6a',
+                formatBalance: '0.000010',
+                isMultisig: true
+            }
+
+            runBasicSetSelectedAccountInfoTests(mockSelectedMultisigAccount);
+        })
+    })
+
+    describe('computeXEMBalance', () => {
+        it('returns formatted XEM balance in string (6 divisibility)', () => {
+            // Arrange:
+            const ctrl = $controller('NodeRewardsProgramCtrl', {
+                $scope: $rootScope.$new()
+            });
+
+            // Act:
+            const balance = ctrl.computeXEMBalance(10);
+
+            // Assert:
+            expect(balance).toEqual('0.000010');
+        })
+    })
+
+    describe('isTabSelected', () => {
+        const runBasicTabSelectedTests = ({selectedTab, tabToTest, expectedResult}) => {
+            it(`returns ${expectedResult} when selected tab ${expectedResult ? 'matched' : 'unmatched'} with current tab`, () => {
+                // Arrange:
+                const ctrl = $controller('NodeRewardsProgramCtrl', {
+                    $scope: $rootScope.$new()
+                });
+
+                ctrl.selectedTab = selectedTab;
+
+                // Act:
+                const resultTab = ctrl.isTabSelected(tabToTest);
```

Comment:
i might have said wrong thing before 😅. but since this is boolean var, it should usually start with is; e.g. something like `isTabToTestSelected`